### ISSUE TITLE
Navigation and Favicon Issue #3 

### ISSUE THREAD

**Issue by arundatin23** [haniicorn/PBJWebsite#4]
As the rubric suggested, the navigation bar should be located in the same place on each page to improve user navigation and accessibility. Moreover, when I go to the main page I noticed that the Favicon is not there. The professor provided ways to add a favicon for our website but I will link it here for you: 
https://favicon.io/

Also don't forget to use a flexbox/ grid layout.
